vendor/modules.txt

```diff
@@ -57,7 +57,7 @@ github.com/cloudflare/circl/math/mlsbset
 github.com/cloudflare/circl/sign
 github.com/cloudflare/circl/sign/ed25519
 github.com/cloudflare/circl/sign/ed448
-# github.com/cockroachdb/cockroach-cloud-sdk-go v1.8.0
+# github.com/cockroachdb/cockroach-cloud-sdk-go v1.8.0 => ../cockroach-cloud-sdk-go
```

Comment:
I think this will need to land before this can be safely comitted.

Reply:
yes... I didn't mean to push this change up 🤦